test/dkanextension/src/Drupal/DKANExtension/Context/LinkcheckerContext.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+// @codingStandardsIgnoreFile
+namespace Drupal\DKANExtension\Context;
+
+use Behat\Behat\Hook\Scope\BeforeFeatureScope;
+use Behat\Behat\Hook\Scope\AfterFeatureScope;
+use Drupal\DKANExtension\Hook\Scope\BeforeDKANEntityCreateScope;
+
+/**
+ * Defines application features from the specific context.
+ */
+class LinkcheckerContext extends RawDKANContext {
+
+  protected $old_global_user;
```

Comment:
At some point we should discuss OOP style standards as a group, but I personally like proper camel case both for methods and properties of a class.

Reply:
k